nextcord/client.py

```diff
@@ -2621,3 +2621,28 @@ def decorator(func: Callable):
             return result
 
         return decorator
+
+    def parse_mentions(self, text: str) -> List[User]:
+        """Parses user mentions in a string and returns a list of :class:`User` objects.
+
+        .. note::
+
+            This does not include role or channel mentions. See :meth:`Guild.parse_mentions` for
+            :class:`Member` objects, :meth:`Guild.parse_role_mentions` for :class:`Role` objects,
+            and :meth:`Guild.parse_channel_mentions` for :class:`~abc.GuildChannel` objects.
+
+        .. versionadded:: 2.2
+
+        Parameters
+        ----------
+        text: :class:`str`
+            String to parse mentions in.
+
+        Returns
+        -------
+        List[:class:`User`]
+            List of :class:`User` objects that were mentioned in the string.
+        """
+
+        it = filter(None, map(self.get_user, utils.parse_raw_mentions(text)))
```

Comment:
What if in the text someone mentions a user object that isn't in the cache?

Reply:
In that event, the developer should use parse_raw_mentions and fetch users as needed.

I don't think the helper function for getting the mentions from text should be making API calls.

This is also the same implementation used already by Message.channel_mentions, etc.

Reply:
Ah that makes sense